haverexample.java

```diff
@@ -0,0 +1,39 @@
+// Java program for the haversine formula 
```

Comment:
The program seems to be directly copied from GeeksForGeeks (https://www.geeksforgeeks.org/haversine-formula-to-find-distance-between-two-points-on-a-sphere/)
Kindly implement this yourself.
@jainaman224